### DISCUSSION TITLE
How to delete blenderbot2_400M and blenderbot1_3B

### DISCUSSION THREAD

**Discussion by Jawad1347** [facebookresearch/ParlAI#3922] [Q&A]
Due to insufficient RAM (was trying to run on a local machine without GPU) the models crash and do not load. And return this response:

> EOFError: Compressed file ended before the end-of-stream marker was reached

I tried to test 3B blenderbot1 on colab without GPU and it failed there too (though response was different, but I was able to see that RAM filled that is why model crashed).

I created an environment and ran 90M and 400Mdistill (blenderbots1) without GPU and I want to continue using them so don't want to delete the environment completely, I just want to remove the particular models and their dependencies (unless they are dependencies of the models that I want to continue using) completely. So, where do I find these files and delete them manually?

**Comment by stephenroller** [comment]
Clear them out of the parlai/data/models folder.

**Reply by superchargez** [reply]
Thank you
